eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -1,137 +1,237 @@
         <Bug pattern="SIC_INNER_SHOULD_BE_STATIC_ANON"/>
-      </Match>
+    </Match>
 
-      <!-- KeyVaultCredential values manipulated do not need to be localised. They
-      are base64 encoded or URL encoded.-->
-      <Match>
+    <!-- KeyVaultCredential values manipulated do not need to be localised. They
+    are base64 encoded or URL encoded.-->
+    <Match>
         <Class name="com.microsoft.azure.keyvault.authentication.KeyVaultCredentials"/>
         <Method name="~(extractChallenge|isValidChallenge|supportsMessageProtection)"/>
         <Bug pattern="DM_CONVERT_CASE"/>
-      </Match>
-
-      <!-- HTTP header, "content-type" is not localised. -->
-      <Match>
+    </Match>
+    <!-- HTTP header, "content-type" is not localised. -->
+    <Match>
         <Class name="com.microsoft.azure.keyvault.messagesecurity.HttpMessageSecurity"/>
         <Method name="unprotectResponse"/>
         <Bug pattern="DM_CONVERT_CASE"/>
-      </Match>
-
-      <!-- HTTP header, "ocp-date" is not localised. -->
-      <Match>
+    </Match>
+    <!-- HTTP header, "ocp-date" is not localised. -->
+    <Match>
         <Class name="com.microsoft.azure.batch.auth.BatchSharedKeyCredentialsInterceptor"/>
         <Method name="signHeader"/>
         <Bug pattern="DM_CONVERT_CASE"/>
-      </Match>
+    </Match>
 
-      <!-- Known issue in autorest where it will generate a variable with a value of null.
-      https://github.com/Azure/autorest.java/issues/324 -->
-      <Match>
+    <!-- Known issue in autorest where it will generate a variable with a value of null.
+    https://github.com/Azure/autorest.java/issues/324 -->
+    <Match>
         <Class name="~com\.microsoft\.azure\.(.+)Impl"/>
         <Bug pattern="NP_LOAD_OF_KNOWN_NULL_VALUE"/>
-      </Match>
+    </Match>
 
-      <!-- Known issue in autorest where it will generate a variable with a value of null then use it in a not null check.
-      https://github.com/Azure/autorest.java/issues/328 -->
-      <Match>
+    <!-- Public field already exists in the public API surface area even though no writes were seen to this public field.-->
+    <!--https://github.com/Azure/azure-sdk-for-java/blob/master/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/BatchOptions.java#L50-->
+    <!--https://github.com/Azure/azure-sdk-for-java/blob/master/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/BatchOptions.java#L57-->
+    <Match>
+        <Class name="~com.microsoft.azure.eventhubs.BatchOptions"/>
+        <Field name="~(maxMessageSize|partitionKey)"/>
+        <Bug pattern="UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD"/>
+    </Match>
+
+    <!-- Public method already exists in the public API surface area even though it has confusing name.-->
+    <!--https://github.com/Azure/azure-sdk-for-java/blob/master/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/MessageReceiver.java#L247-->
```

Comment:
Comments can span multi lines, you can use something like:
```xml
<!-- Public method already exists in the public API surface area even though it has confusing name.
  https://github.com/Azure/azure-sdk-for-java/blob/master/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/MessageReceiver.java#L251
  https://github.com/Azure/azure-sdk-for-java/blob/master/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/MessagingFactory.java#L146
  https://github.com/Azure/azure-sdk-for-java/blob/master/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/MessageReceiver.java#L247-->
```

Reply:
thanks